projects/plugins/jetpack/_inc/client/recommendations/sidebar/layout/index.jsx

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import './style.scss';
 
 const Layout = props => {
```

Comment:
The name `Layout` got me confused a bit here, since the component actually represents a card (and we've got 2 now). Maybe it could be a good opportunity to rename it, what do you think?

Reply:
I think it's a great opportunity indeed. What do you think about `SidebarCard`? So to actually match class name?

When we're renaming the component I also thought about one more improvement - should we accept `content` as `children` rather than a custom property? It would make usage of the component clearer i.e.:

```
<SidebarCard>{children}</SidebarCard>
```
instead of
```
<SidebarCard content={children} />
```

as the content is quite extensive in all cases.